Calendar/Get-RBASummary.ps1

```diff
@@ -522,6 +540,52 @@ function RBALogSummary {
         if ($AcceptLogs.count -eq 0 -and $TentativeLogs.count -eq 0 -and $DeclineLogs.count -eq 0) {
             Write-Host -ForegroundColor Red "`t No meetings were processed in the RBA Log."
         }
+
+        if ($UpdatedLogs.count -ne 0) {
+            $LastUpdated = ($UpdatedLogs[0] -Split ",")[0].Trim()
+            Write-Host "`t $($UpdatedLogs.count) Updates to meetings between $FirstDate and $LastDate"
```

Comment:
It is possible that `$FirstDate` is empty, because it is only set inside of a `if` statement.